packages/airnode-node/src/constants.ts

```diff
@@ -31,10 +31,10 @@ export const DEFAULT_RETRY_DELAY_MS = 50;
 export const EVM_PROVIDER_TIMEOUT = 10_000;
 
 // The maximum amount of time the "initialize provider" worker is allowed before being timed out
-export const WORKER_PROVIDER_INITIALIZATION_TIMEOUT = 19_500;
+export const WORKER_PROVIDER_INITIALIZATION_TIMEOUT = 11_500;
```

Comment:
I wonder if we need these 500ms buffers. I originally added them to allow the worker process some time to return a response (even if it's null) instead of having the entire worker promise reject, but I'm not sure this is necessary.

Reply:
I'm struggling to find the exact logs now (to quote them) but early on in my testing, before I increased the Terraform timeouts, I encountered quite a few timeouts.

From memory, when the handlers fail due to a Terraform-related timeout, `startCoordinator` emits a JSON unmarshalling error message, along the lines of "bad character 'u' at position x", which I suspect is "undefined" underneath.

Still, the requests got through and the chain is the ultimate reference, so you're probably right, the timing guards are probably unnecessary, but will result in cryptic error messages if the timeouts get hit. I can test that as a scenario quick and get back to you if you want?

Reply:
Yeah sure - that sounds good 👍🏻

Reply:
I'll get round to this - I've just been getting the stress tester working with GCP (so far so good (!) just have to implement GCP log grabbing and processing).

Reply:
@andreogle 
This is how `initializeProvider` times out without buffers/when the Terraform timeout gets hit:
```
3c3a2f91-f062-4ebc-b733-a80726e5eaf5	ERROR	Uncaught Exception 	{
    "errorType": "SyntaxError",
    "errorMessage": "Unexpected token u in JSON at position 0",
    "code": "SyntaxError",
    "message": "Unexpected token u in JSON at position 0",
    "time": "2021-11-26T12:14:31.926Z",
    "stack": [
        "SyntaxError: Unexpected token u in JSON at position 0",
        "    at JSON.parse (<anonymous>)",
        "    at constructor.<anonymous> (/var/task/handlers/aws/index.js:2:1972298)",
        "    at constructor.<anonymous> (/var/task/handlers/aws/index.js:2:711892)",
        "    at constructor.callListeners (/var/task/handlers/aws/index.js:2:729208)",
        "    at constructor.emit (/var/task/handlers/aws/index.js:2:728918)",
        "    at constructor.emitEvent (/var/task/handlers/aws/index.js:2:714767)",
        "    at constructor.e (/var/task/handlers/aws/index.js:2:710272)",
        "    at t.runTo (/var/task/handlers/aws/index.js:2:792563)",
        "    at /var/task/handlers/aws/index.js:2:792769",
        "    at constructor.<anonymous> (/var/task/handlers/aws/index.js:2:710542)"
    ]
}
[ERROR] [1637928871988] LAMBDA_RUNTIME Failed to post handler success response. Http response code: 400.
```
and with the timeout hitting the worker options value from `constants.ts`:
```
3ffa77b0-0531-4fb5-9d94-0f7a1cbfe695	INFO	[2021-11-26 12:33:16.498] ERROR Error: Operation timed out in 2000 ms.
    at Timeout._onTimeout (/var/task/handlers/aws/index.js:2:1970597)
    at listOnTimeout (internal/timers.js:557:17)
    at processTimers (internal/timers.js:500:7)  Coordinator-ID:fc1d5ca1cafb7d60
```
`processProviderRequests` behaves similarly - so it appears there are no problems associated with removing the timing buffers.

Reply:
Ok awesome. I'm happy then if you want to remove them here then. It will also be more intuitive for future readers